### PR TITLE
Build from CI cloned repo (and fix netlify previews)

### PR DESCRIPTION
**Pull request by fhennig**
This PR changes the production build to build from the locally fetched repo in the CI.

As the build happens in netlify, a script is necessary to fetch all to content - netlify does a bare clone initially.

As a consequence the netlify previews in PRs now actually work!

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+[build]
+  command = "bash scripts/netlify_git_fetch.sh && make ANTORAFLAGS=--fetch PLAYBOOK=antora-playbook.yml"
```
Comment (razvan) [inline netlify.toml:2]: Why not  put the content of the script in the Makefile ?

**Review by razvan — COMMENTED** [review]
one question

File: netlify.toml (hunk 1 above)
Comment (fhennig) [reply netlify.toml:2]: Good question. I'll try to do that, would be cleaner

Comment (fhennig) [reply netlify.toml]: I did it!

**Review by razvan — APPROVED** [review]
nice

**Comment by fhennig** [comment]
lovely jubbly!

**Pull request merged**